jetson/percep/pcl.cpp

```diff
@@ -186,177 +186,81 @@ void PCL::FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices,
 
 }
 
+/* --- Get Angle --- */
+//This function finds the angle off the x axis the
+//line that passes through both these points is
+//Direction of 0 is left and 1 is right
+double PCL::getAngle(int buffer, int direction, std::vector<std::vector<int>> &interest_points,
+                    shared_ptr<pcl::visualization::PCLVisualizer> viewer, std::vector<int> &obstacles){
+    double newAngle = 0;
+    //If Center Path is blocked check left until have to turn too far
+    while(newAngle > -70 && newAngle < 70){
+        
+        //Finding angle off center
+        double xDist = pt_cloud_ptr->points[obstacles.at(direction)].x;
+        double zDist = pt_cloud_ptr->points[obstacles.at(0)].z;//Length of adjacent
+        xDist += direction ? buffer+HALF_ROVER : -(buffer+HALF_ROVER); //Calculate length of opposite
+        newAngle = atan(xDist/zDist)*180/PI;//tan-1(opposite/adjacent)
+
+        //Create compareLine Functors
+        compareLine leftLine (newAngle, -HALF_ROVER);
```

Comment:
NIT: remove space after `leftLine` to make it clear what this is doing. Do this throughout.

Reply:
got it